### PR TITLE
Example: creates rhi data from one field and contours from another

### PR DESCRIPTION
**Pull request by Rumpkie**

### CODE REVIEW

File: examples/plotting/plot_rhi_data_overlay.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+display.plot('velocity', sweep=sweep, vmin=-20, vmax=20.0, fig=fig,
+             ax=ax, cmap='pyart_BuDRd18', colorbar_label='Velocity (m/s)')
+
+# line commented out to show reflectivity
+# display.plot('reflectivity', sweep=sweep, vmin=-0, vmax=45.0, fig=fig,ax=ax)
+
+# get data
+start = radar.get_start(sweep)
+end = radar.get_end(sweep) + 1
+data = radar.get_field(sweep,'reflectivity')
```
Comment (jjhelmus) [inline examples/plotting/plot_rhi_data_overlay.py]: Need a space after the comma

File: examples/plotting/plot_rhi_data_overlay.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+x,y,z = radar.get_gate_x_y_z(sweep,edges=False)
```
Comment (jjhelmus) [inline examples/plotting/plot_rhi_data_overlay.py]: spaces needed after the commas

**Comment by jjhelmus** [comment]
Looks good @Rumpkie, just need to fix the two style issues.  We try to follow the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guide in Py-ART.  I use the [pep8 tool](https://pypi.python.org/pypi/pep8) tool to check style before I check files in.  

The 034142.mdv file should also be added to the `.gitignore` file so that it can be placed in the examples/plotting directory without causing issue.

**Comment by jjhelmus** [comment]
Link to issue #450 which this PR address.

**Comment by Rumpkie** [comment]
I made those changes, please try again.

File: .gitignore [hunk 3]
```diff
@@ -124,6 +124,7 @@ examples/plotting/20110520100000_nex_3d.nc
 examples/plotting/20110520100000_nexrad_grid.nc
 examples/plotting/narr-a_221_20110520_0000_000.nc
 examples/plotting/nsaxsaprppiC1.a1.20140201.184802.nc
+examples/plotting/034142.mdv
```
Comment (jjhelmus) [inline .gitignore]: This line has trailing whitespace which for some reason means the file is not ignored.  Can you remove it?

**Comment by jjhelmus** [comment]
Thanks for the updates, for some reason the trailing whitespace in the .gitignore file is causing an issue.  Fix that and this should be good to go.

**Comment by Rumpkie** [comment]
Done

**Comment by jjhelmus** [comment]
Great work!  Merging.

**Pull request merged**